src/quo2/components/drawers/drawer_top/view.cljs

```diff
@@ -37,7 +37,7 @@
    {:size   :paragraph-2
     :weight :regular
     :style  (style/network-text-color network)}
-   (str (subs (name network) 0 3) ":")])
+   (str (subs (name network) 0 3) (when (= network :arbitrum) "1") ":")])
```

Comment:
hmm do we want to couple the network to the component this much? what happens when we are using test networks etc? 🤔

Reply:
@smohamedjavid just checked with design team, it should be `arb1`